.circleci/continue_config.yml

```diff
@@ -444,6 +454,29 @@ workflows:
           docker_name: op-signer
           docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
           docker_context: .
+  peer-mgmt-service:
+    when:
+      or: [<< pipeline.parameters.run-build-pms >>, << pipeline.parameters.run-all >>]
+    jobs:
+      - go-lint:
+          name: pms-lint
+          module: peer-mgmt-service
+      - go-test:
+          name: pms-tests
+          module: peer-mgmt-service
+      - docker-build:
+          name: pms-docker-build
+          docker_file: peer-mgmt-service/Dockerfile
+          docker_name: peer-mgmt-service
+          docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
+          docker_context: .
+      - docker-build:
+          name: pms-docker-build-backwards-compatible
+          docker_file: peer-mgmt-service/Dockerfile
+          docker_name: peer-mgmt-service
+          docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
+          repo: oplabs-tools-artifacts/internal-images ## introduces backwards compatibility
```

Comment:
not sure the backwards compatibility is necessary imo.. we should just do a fast follow image update after we merge this

Reply:
> not sure the backwards compatibility is necessary imo.. we should just do a fast follow image update after we merge this

Sure I don't mind, I personally kept it as a safety measure in case we were to potentially release a quick hotfix to PMS in this repo while the Deployments in our clusters still pointed to the private images. (as a failsafe).

> Could go ahead and create a draft pr against the private repo to remove it there. Would be good to remove it shortly after this pr is merged to avoid confusion around multiple sources for pms images

Already done [here](https://github.com/ethereum-optimism/infrastructure-services/pull/308) : )

Reply:
Yeah makes sense.. anyways I doubt we will have any hotfix / new changes to this any time soon. Will leave it up to you.